frontend/packages/ceph-storage-plugin/integration-tests/tests/2-tests/ocp-dashboard-card-healthcheck.scenario.ts

```diff
@@ -0,0 +1,75 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '@console/internal-integration-tests/protractor.conf';
+import { dashboardIsLoaded } from '@console/shared/src/test-views/dashboard-shared.view';
+import { click } from '@console/shared/src/test-utils/utils';
+import {
+  mainHealtGreenSvg,
+  mainHealthCardStatus,
+  seeAllLink,
+  seeAllClose,
+  seeAllStorageGreenSvg,
+  seeAllStorageSection,
+  noOutChange,
+  mainHealtYellowSvg,
+  seeAllStorageYellowSvg,
+  mainHealtRedSvg,
+  seeAllStorageRedSvg,
```

Comment:
@shyRozen Please sort the imports alphabetically. Its a standard followed by console team.

Reply:
Ok.Thanks

Reply:
Fixed.